Added SteamID Support.

kratos1812 opened the pull request:
Now you can limit the acces to a Custom MVP to a single steamid. Example in kento_mvp.cfg. I've seen many people requesting this.

kratos1812 committed:
Added SteamID Support.

Now you can limit the acces to a Custom MVP to a single steamid. Example in kento_mvp.cfg

kratos1812 committed:
Fixed an menu mistake in the newest update

rogeraabbccdd commented:
Looks like this PR only support 1 SteamID.
I think multiple SteamID support would be better.

kratos1812 committed:
Added multipele steamids support

Now each Custom MVP supports multiple steamids at once. Example in kento_mvp.cfg

kratos1812 commented:
> Looks like this PR only support 1 SteamID.
> I think multiple SteamID support would be better.

Just pushed a new commit. I tested it and works well(debuged it, etc etc).
I've added a new cvar too `mvp_no_acces_show_mode`.

rogeraabbccdd commented on addons/sourcemod/scripting/kento_mvp.sp:
```diff
@@ -276,9 +314,31 @@ void DisplayMVPMenu(int client, int start)
+				}
+				else
+				{
+					switch(Cvar_ShowMode.IntValue)
```
I think hook cvar change and store value in a variable is better than calling `.IntValue` everytime.

kratos1812 committed:
Removed the usage of Cvar_ShowMode.IntValue

As Kento requested.

rogeraabbccdd commented:
LGTM
